### ISSUE TITLE
about inplace_abn and inplace_abn_sync 

### ISSUE THREAD

**Issue by JamesKasperYu** [mapillary/inplace_abn#43]
I used inplace_abn in my net  and  got a best model,then I just change the norm_act=InplaceABNSync to finetune the best model,but the miou less 1-2%.
I used inplace_abn and inplace_abn_sync in teh same net,the after also less 1%.I trained in 2 TITAN GPUs with batchsize = 16.Why I got a  worse result for inplace_abn_sync?Thank you!

**Comment by rotabulo** [comment]
@JamesKasperYu InplaceABNSync provides a better estimate of the BN statistics, due to the larger sample size. Hence, the variance of the gradient reduces and the method should exhibit better convergence. This also means that you might eventually reduce the implicit regularization effect due to the higher variance in the gradients and, hence, worsen the results in case of small training sets, more prone to overfitting. Before considering this hypothesis, however, I would also try to either train from scratch with sync on, or perform the same finetuning with InplaceABN (non-sync). Maybe it's just the finetuning per se that degrades the performance.

**Comment by JamesKasperYu** [comment]
Indeed, the train miou of InplaceABNSync is better than of InplaceABN in a experiment whose sets  are  the start learning rate divided 2,change 4gpus with batchsize=16 from 1gpu with batchsize=5.Other sets are same.Did the start learning rate affect the result.Or what can I do to get a better val miou by use InplaceABNSync finetuning the net.

**Comment by JamesKasperYu** [comment]
I have tried keeping the learning rate to finetune with sync and got a better result.Thank you!
